lib/runners/processor/pylint.rb

```diff
@@ -0,0 +1,93 @@
+module Runners
+  class Processor::Pylint < Processor
+    include Python
+
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+          target: enum?(string, array(string)),
+          rcfile: string?,
+          ignore: enum?(string, array(string)),
+          'errors-only': boolean?,
+        })
+      }
+
+      let :rule, object(
+        severity: string,
+      )
+    end
+
+    register_config_schema(name: :pylint, schema: Schema.runner_config)
+
+    DEFAULT_TARGET = ["**/*.{py}"].freeze
+
+    def analyze(changes)
+      run_analyzer
+    end
+
+    private
+
+    def rcfile
+      file = config_linter[:rcfile]
+      Array(file ? "--rcfile=#{file}" : nil)
+    end
+
+    def ignore
+      Array(config_linter[:ignore] || []).then { |arr| arr.empty? ? nil : "--ignore=#{arr.join(",")}" }
+    end
+
+    def erros_only
+      config = config_linter[:'errors-only'] || config_linter.dig(:options, :'errors-only')
+      ["--errors-only"] if config
```

Comment:
```suggestion
      config_linter[:'errors-only'].then { |value| value ? ["--errors-only"] : [] }
```

`:options` does not exist 😏